docker/ravendb-ubuntu/cert-utils.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+function extract_cert_and_key() {
```

Comment:
here's underscores and in other places it's spaces?

Reply:
done